modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -28,6 +28,15 @@ definition {
 				userEmailAddress = "partner@liferay.com");
 		}
 
+		task ("Add Manager Channel role to the user") {
```

Comment:
Hi @mateussandes23, update this task do 'Add a new user with a Manager role'

Reply:
Also, update the [commit message](https://github.com/pt-liferay-solutions/liferay-portal/pull/3228/commits/27d75eb4e51c19d8611244d878e143db430b1c4d#)